src/core/services/theming/theming.spec.js

```diff
@@ -754,10 +754,14 @@ describe('$mdTheming service', function() {
 
     $mdTheming.defineTheme('test', {
       primary: 'red',
+      primaryHues: {
+        default: '300'
+      },
       warn: 'yellow'
     });
 
     expect($mdTheming.THEMES.hasOwnProperty('test')).toBeTruthy();
+    expect($mdTheming.THEMES.test.colors.primary.hues.default).toBe('300');
```

Comment:
I suppose this should be enough to test the hue overrides. Suggestions?

Reply:
The assertion looks like it will work.

However, it would be helpful if, instead of changing this test, you could add a new test that was similar but set and verified each of the different `xHues` fields.

Reply:
I did a new test case, following your suggestion. I agree that this is a better way of testing it.